library-best-practices/JLBP-14.md

```diff
@@ -0,0 +1,11 @@
+[JLBP-14] Use explicit versions, not version ranges
+---------------------------------------------------
+
+- Neither Maven nor Gradle have have a mechanism to locally lock versions like
```

Comment:
Is the first sentence needed?

Reply:
No it's honestly not needed, since the JLBPs are focused on Java library authors. I will remove it.